src/OrbitDB.js

```diff
@@ -210,10 +240,10 @@ let databaseTypes = {
 
     const getStore = address => this.stores[address]
     const getDirectConnection = peer => this._directConnections[peer]
-    const onChannelCreated = channel => this._directConnections[channel._receiverID] = channel
+    const onChannelCreated = channel => this._directConnections[channel._receiverID] = channel // eslint-disable-line
```

Comment:
Nit-picking, but it'd be generally better to fix the code style rather than adding // eslint-disable.line. Here, iirc (I may not), the fixed style is:
```js
const onChannelCreated = channel => (this._directConnections[channel._receiverID] = channel)
```

Reply:
Good call. I simply removed the annotation in this branch since I'm doing all the linting in `fix/persisent-keystore-linting`. This one simply snuck in. I also removed the annotation from the other branch as well so that I remember to fix it :)